test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1444,7 +1445,6 @@ def test_da8w8_sym_act_sym_wgt_with_int_mm(
         """
         if dtype == torch.bfloat16 and not torch.ops.mkldnn._is_mkldnn_bf16_supported():
             return
-        M = 32
         in_feature = 48
         out_feature = 64
         q_min, q_max = -32, 31
```

Comment:
It seems this unit test doesn't verify whether the CPP Template has been used. Can we add a counter check similar to https://github.com/pytorch/pytorch/pull/147895?

Reply:
I could rebase this PR when #147895 will be merged (but it seems like its merge procedure ran into some CI issue), so I'll add it now.